web_responsive/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "name": "Web Responsive",
     "summary": "It provides a mobile compliant interface for Odoo Community "
                "web",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.0.1",
```

Comment:
Are you sure? I think that's not needed to update a client-side qweb template, isn't it?